src/pcrlock/pcrlock.c

```diff
@@ -2101,7 +2102,7 @@ static int show_log_table(EventLog *el, JsonVariant **ret_variant) {
                         if (bank) {
                                 _cleanup_free_ char *hex = NULL;
 
-                                hex = hexmem(bank->hash.buffer, bank->hash.size);
+                                hex = hexmem(bank->hash.buffer, arg_full?bank->hash.size:5);
```

Comment:
can you move this to a helper so that the "5" is not repeated over and over again?

by coding style we insert spaces before/after ? and : btw...

maybe a helper:

```c
static char *hexmem_abbreviate(const void *buf, size_t size) {
        return hexmem(buf, arg_full ? size : MIN(size, 5));
}
```

git abbeviates to 11 chars, not 12, don't they? should we do so too?

Reply:
AFAICT in git the fallback is 7 hex digits. Don't know why I picked 5 here exactly, probably just to get a feeling whether that kind of output is acceptable. Will refactor.